rasa/nlu/utils/pattern_utils.py

```diff
@@ -21,13 +22,16 @@ def _convert_lookup_tables_to_regex(
     for table in training_data.lookup_tables:
         if use_only_entities and table["name"] not in training_data.entities:
             continue
-        regex_pattern = _generate_lookup_regex(table)
+        regex_pattern = _generate_lookup_regex(table,config_language=config_language)
         lookup_regex = {"name": table["name"], "pattern": regex_pattern}
         patterns.append(lookup_regex)
     return patterns
 
 
-def _generate_lookup_regex(lookup_table: Dict[Text, Union[Text, List[Text]]]) -> Text:
+def _generate_lookup_regex(
+        lookup_table: Dict[Text, Union[Text, List[Text]]],
+        config_language: Text = None,
```

Comment:
Each config must have a language, I can't think of a time where this would be `None`, can you? If we provide `None` for this, we'll get an exception when we try to `lower()` it.

Reply:
I am a newer in python, so i don't know what will happen and how it work, i will listen to your suggestions